### PR TITLE
Minimal mode

### LINKED ISSUE #59: "Minimal mode" idea
I was thinking about possibly adding a "minimal mode" option that would hide certain elements on the page such as the right side panel, the support question mark, the footer information, keyboard button, etc.

@timche Thoughts on adding this as an option?

### PR DESCRIPTION
**Pull request by markypython**
Adds a new minimal mode to clean up some of the messy Gmail interface.  Fixes #59

### CODE REVIEW

File: src/utils.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { BrowserWindow } from 'electron'
+
+export function getWindow(): BrowserWindow {
+  return BrowserWindow.getAllWindows()[0]
```
Comment (timche) [inline src/utils.ts]: The first window is the main window where `gmail.com` is loaded right? Should we name the function `getMainWindow()` instead?

File: src/utils.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+}
+
+export function sendAction(action: string, ...args: unknown[]): void {
```
Comment (timche) [inline src/utils.ts]: How about naming it more specific like `sendChannelToMainWindow()` to use the same term as in the [doc](https://electronjs.org/docs/api/web-contents#contentssendchannel-arg1-arg2-)?

**Comment by timche** [comment]
Please merge this PR rebased to keep the commits and don't squash them as this PR does multiple things, not only adding the minimal mode.

**Comment by timche** [comment]
I'm excited for the new update 😃

File: src/utils.ts (hunk 1 above)
Comment (markypython) [reply src/utils.ts]: Yes, the first window is the one loaded when the app starts.  I will rename the function

File: src/utils.ts (hunk 2 above)
Comment (markypython) [reply src/utils.ts]: Updated!

**Pull request merged**